cloudflare-mixin/dashboards/cloudflare-worker-overview.libsonnet

```diff
@@ -0,0 +1,402 @@
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisBorderShow: false,
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 10,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'linear',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+{
+  grafanaDashboards+:: {
+    'cloudflare-worker-overview.json':
+      dashboard.new(
+        'Cloudflare worker overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        description='',
+        uid=dashboardUid,
+      )
+
+      .addLink(grafana.link.dashboards(
+        asDropdown=false,
+        title='Other Cloudflare dashboards',
+        includeVars=true,
+        keepTime=true,
+        tags=($._config.dashboardTags),
+      ))
+
+      .addTemplates(
+        [
+          template.datasource(
+            promDatasourceName,
+            'prometheus',
+            null,
+            label='Data Source',
+            refresh='load'
+          ),
+          template.new(
+            'job',
+            promDatasource,
+            'label_values(cloudflare_zone_requests_total,job)',
+            label='Job',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='.+',
+            sort=0
+          ),
+          template.new(
+            'instance',
+            promDatasource,
+            'label_values(cloudflare_zone_requests_total{job="$job"},instance)',
+            label='Instance',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='.+',
+            sort=0
+          ),
+          template.new(
+            'script_name',
+            promDatasource,
+            'label_values(cloudflare_worker_requests_count{instance="$instance"},script_name)',
```

Comment:
Isn't that accounted for with the instance filter since the instance filter has a job filter? 

Side note: I did use `=` instead of `=~` I noticed and changed that.

Reply:
If we use `.+` regex (we normally do) for allValue, then it could be that instance filter would be `instance=~".+"` so ignoring any filtering done in instance variable.